rules/windows/file/file_event/file_event_win_create_evtx_non_common_locations.yml

```diff
@@ -22,6 +22,7 @@ detection:
         TargetFilename|endswith: '.evtx'
     filter_main_path:
         TargetFilename|startswith: 'C:\Windows\System32\winevt\Logs\'
+        TargetFilename|startswith: 'C:\Windows\System32\winevt\EventLogs'
```

Comment:
I've never seen this. Mind sharing a reference? I feel like this is some GPT hallucination. Did you use AI to get this?